playwright/tests/slices/slicesList.spec.ts

```diff
@@ -48,6 +48,23 @@ test.describe("Slices list", () => {
     },
   );
 
+  test.run()(
+    "I cannot create a slice with a name that already exists",
+    async ({ slicesListPage, slice }) => {
+      await slicesListPage.goto();
+      await slicesListPage.openCreateDialog();
+
+      const { nameInput, submitButton } = slicesListPage.createSliceDialog;
+
+      await nameInput.fill(slice.name);
+      await expect(submitButton).toBeDisabled();
+      await submitButton.click({ force: true });
```

Comment:
Rather than forcing a button click, should we test for form submission (e.g. pressing Enter)?

Reply:
That was a good idea, BUT sadly again, the way form management is done is that the error message is only displayed after a click outside of the input --"
This will need to be redone with new form management.